hermes/bin/src/packaging/mod.rs

```diff
@@ -93,6 +94,12 @@ fn copy_resource_dir_recursively_to_package(
     errors.return_result(())
 }
 
+/// Remove file or the whole directory from the package.
+fn remove_item_from_package(name: &str, package: &hdf5::Group) -> anyhow::Result<()> {
```

Comment:
We discussed offline with @Mr-Leshiy to refactor `remove_item_from_package` to separate functions
`remove_file_from_package` and `remove_dir_from_package` with appropriate checks etc. 

Here is some sudo code which we discussed:

```rust
fn remove_item_from_package(name: &str, package: &Group) -> Result<()> {
    let item = package
        .obj(name)
        .with_context(|| format!("Failed to find '{}' in the package", name))?;

    if item.is_group() {
        info!("Removing directory '{}' from the package", name);
    } else if item.is_dataset() {
        info!("Removing file '{}' from the package", name);
    } else {
        return Err(anyhow::anyhow!("Unsupported item type for '{}'", name));
    }

    package
        .unlink(name)
        .with_context(|| format!("Failed to remove '{}' from the package", name))?;
    info!("Successfully removed '{}' from the package", name);
    Ok(())
}
```